bundle/manifests/authorino-operator.clusterserviceversion.yaml

```diff
@@ -8,32 +8,25 @@ metadata:
           "apiVersion": "operator.authorino.kuadrant.io/v1beta1",
           "kind": "Authorino",
           "metadata": {
-            "name": "authorino-sample",
-            "namespace": "authorino-operator"
+            "name": "authorino-sample"
           },
           "spec": {
-            "clusterWide": true,
-            "image": "quay.io/kuadrant/authorino:latest",
-            "imagePullPolicy": "Always",
             "listener": {
-              "port": null,
               "tls": {
                 "certSecretRef": {
-                  "name": "authorino-cert"
+                  "name": "authorino-server-cert"
                 },
                 "enabled": true
```

Comment:
Do we want `enabled: false` here, make it even smaller without `certSecretRef` and nothing else required to try the example, or would that be sending a wrong message that we recommend TLS to be disabled, which we obviously do not?

In the opposite direction, `enabled: true` could be omitted here. But could that impose an obstacle to users who want to try without TLS first (e.g. for non-prod env) but think the only way is by creating the TLS secrets?

Reply:
I'd say `"enabled": false"`. Thus the config not only is smaller but also communicates to the user the setting exists.

Reply:
Done